internal/executors/http/tls/tls_configuration_provider.go

```diff
@@ -125,29 +117,19 @@ func (p *TLSConfigurationProvider) trustCertificate(tr *http.Transport, certs st
 	return nil
 }
 
-func decodeIfBase64Certificate(certs string, errMessage string) (string, error) {
-	if !isBase64(certs) {
-		return certs, nil
-	}
-
-	d, err := base64.StdEncoding.DecodeString(certs)
+func decodeIfBase64Certificate(certs string) string {
+	decoded, err := base64.StdEncoding.DecodeString(certs)
 	if err != nil {
-		return "", executors.NewNonRetryableError(errMessage)
+		return certs
 	}
-
-	return string(d), nil
-}
-
-func isBase64(s string) bool {
-	return regexp.MustCompile(BASE64_ENCODING_PATTERN).MatchString(s)
+	return string(decoded)
 }
```

Comment:
Make the function return `[]byte` instead as all usages end up casting it back to `[]byte` in order to use it anyways. :)